packages/apps/src/Menu/Grouping.tsx

```diff
@@ -6,90 +6,228 @@ import { Group } from './types';
+import { Route } from '@polkadot/apps-routing/types';
 
 interface Props extends Group {
   className?: string;
+  variant?: string;
+  activeRoute?: Route;
 }
 
 const SHA_COL = 'rgba(34, 36, 38, 0.12)';
 const SHA_OFF = '5px';
 
-function Grouping ({ className = '', name, routes }: Props): React.ReactElement<Props> {
-  if (routes.length === 1) {
-    return (
-      <Item
-        isToplevel
-        route={routes[0]}
-      />
-    );
-  }
+function Grouping ({ activeRoute, className = '', name, routes, variant }: Props): React.ReactElement<Props> {
+  if (routes && routes.length === 1) {
+    switch (variant) {
+      case 'active-tab': {
+        return (
+          <>{activeRoute &&
+            <MenuActiveItem className='highlight--color'>
+              <Icon icon={activeRoute.icon} />
+              <span>{activeRoute.text}</span>
+            </MenuActiveItem>}
+          </>
+        );
+      }
 
-  return (
-    <li className={className}>
-      <div className='groupHdr highlight--color-contrast'>
-        <span>{name}</span>
-        <Icon icon='caret-down' />
-      </div>
-      <ul className='groupMenu highlight--bg-light'>
-        {routes.map((route): React.ReactNode => (
+      default: {
+        return (
           <Item
-            key={route.name}
-            route={route}
+            isToplevel
+            route={routes[0]}
           />
-        ))}
-      </ul>
-    </li>
-  );
+        );
+      }
+    }
+  }
+
+  switch (variant) {
+    case 'active-tab': {
+      return (
+        <li className={className}>
+          {activeRoute &&
+          <MenuActiveItem className='highlight--color'>
+            <Icon icon={activeRoute.icon} />
+            <span>{activeRoute.text}</span>
+            {routes && routes.length > 1 &&
+              <Icon
+                className='dropdown'
+                icon={'caret-down'}
+              />
+            }
+          </MenuActiveItem>}
+          {routes && routes.length > 1 &&
+            <ul className='groupMenu tab highlight--bg-light'>
+              {routes.map((route): React.ReactNode => {
+                const itemClassName = (activeRoute && activeRoute.text === route.text) ? 'active highlight--color' : '';
+
+                return <Item
+                  className={itemClassName}
+                  key={route.name}
+                  route={route}
+                />;
+              })}
+            </ul>
+          }
+        </li>
+      );
+    }
+
+    default: {
+      return (
+        <li className={className}>
+          <div className='groupHdr highlight--color-contrast'>
+            <span>{name}</span>
+          </div>
+          <ul className='groupMenu highlight--bg-light'>
+            {routes.map((route): React.ReactNode => (
+              <Item
+                key={route.name}
+                route={route}
+              />
+            ))}
+          </ul>
+        </li>
+      );
+    }
+  }
 }
 
+const MenuActiveItem = styled.div`
```

Comment:
Please don't use this form in the apps codebase. Which this is the recommended form from styled-components, it is hell on maintenance. (Where it is used elsewhere, e.g. the extension, these is an active effort to remove all - some are a nightmare to see what is local, what is external, etc.)